rust/oak_derive/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "oak_derive"
+version = "0.1.0"
+authors = ["Tiziano Santoro <tzn@google.com>"]
+edition = "2018"
```

Comment:
Not 2019?  :)

Reply:
The only editions are 2015 and 2018, for now :)
https://doc.rust-lang.org/edition-guide/rust-2018/index.html